src/Serilog.FullNetFx/LoggerConfigurationFullNetFxExtensions.cs

```diff
@@ -267,12 +267,19 @@ public static class LoggerConfigurationFullNetFxExtensions
         /// <code>serilog:minimum-level</code>.
         /// </summary>
         /// <param name="settingConfiguration">Logger setting configuration</param>
+        /// <param name="settingPrefix">Optional prefix to use when reading keys in appSettings</param>
         /// <returns>An object allowing configuration to continue.</returns>
         public static LoggerConfiguration AppSettings(
-            this LoggerSettingsConfiguration settingConfiguration)
+            this LoggerSettingsConfiguration settingConfiguration, string settingPrefix = null)
```

Comment:
For v1.5, can we use an overload and non-default param so that the change is non-breaking? Otherwise all looks great, thanks for taking the time to push this one forwards.